ui/src/components/flows/Flows.vue

```diff
@@ -801,17 +805,6 @@
                     this.loadData(() => {});
                 });
             },
-            chartData(row) {
```

Comment:
There's a console error reporting this is missing in the new version on `Flows` page.